src/debug/di/module.cpp

```diff
@@ -880,6 +881,11 @@ HRESULT CordbModule::InitPublicMetaDataFromFile(const WCHAR * pszFullPathName,
                                                 DWORD dwOpenFlags,
                                                 bool validateFileInfo)
 {
+#ifdef FEATURE_PAL    
+    // TODO: Some intricate details of file mapping don't work on Linux as on Windows.
```

Comment:
Just a nit - we tend to us UNIXTODO instead of TODO for Linux / Darwin / ... specific issues